src/AvtaleSide/steg/BeregningTilskudd/BeregningTilskuddSteg.less

```diff
@@ -0,0 +1,5 @@
+.beregningTilskuddSteg {
+    &__luft {
+        padding: 0.5rem 0 0.5rem 0;
```

Comment:
Kan skrives slik: `padding: 0.5rem 0;`
Luft er forøvrig ikke et særlig bra klassenavn, bør heller være beskrivende for elementet det brukes på.
Synes kanskje det er litt lite luft mellom ting generelt på denne siden, men der kan Preben få siste ord 🤓

Reply:
Enig med deg at luft ikke er bra klassenavn men tenkte å bruke den for flere komponenter, men det kanskje jeg har ikke gjort hehe. Kom gjerne med forslag :)

Reply:
Ok, men nå er navnet på den klassen relativt spesifikk for denne komponenten, siden den heter `beregningTilskuddSteg__luft` . Den kunne kanskje hett noe relatert til elementene den brukes på, eks `tittel`. Alternativt kan man også brukes `<VerticalSpacer sixteenPx="true" />` også 🙂